Style-guide/README.md

```diff
@@ -81,39 +78,24 @@ Descriptive and consistent naming makes software easier to read and understand.
 - naming methods for their side effects
   - verb methods follow the -ed, -ing rule for the non-mutating version
   - noun methods follow the formX rule for the mutating version
-  - boolean types should read like assertions
+  - boolean types should read like assertions (`isLoading`, `isHidden`, etc)
   - protocols that describe _what something is_ should read as nouns
   - protocols that describe _a capability_ should end in _-able_ or _-ible_
 - using terms that don't surprise experts or confuse beginners
-- generally avoiding abbreviations
-- using precedent for names
+- generally avoiding abbreviations // a few well-known ones are allowed, such as, `Html` and `Id`. Any others?
```

Comment:
(I've had to change the line above to trigger a diff, else we couldn't comment on the line. The inline comment will be removed.)

A few well-known ones are allowed, such as, `Html` and `Id`. Any others?

Reply:
is HTML an abbreviation or an acronym?

Reply:
Acronym. HyperText Markup Language.

Reply:
>  Any others?

URL, HTTP. I think the rule of thumb could be if abbreviation already appears in iOS SDK or stdlib, it's ok to use it. Swift Guidelines have pretty nice tip for this:

> The intended meaning for any abbreviation you use should be easily found by a web search.